src/deskolem.ml

```diff
@@ -3,186 +3,216 @@ open! Lplib
 open Term
 open Timed
 
+exception Not_Proof of term
+exception Not_ProofS of term
+
+(** Builtin configuration. *)
+type config =
+  { symb_Skolem     : sym
+  ; symb_Axiom      : sym
+  ; symb_Formula    : sym
+  ; symb_imp        : sym
+  ; symb_forall     : sym
+  ; symb_proof      : sym }
+
+(** [get_config sign] build the configuration using [sign]. *)
+let get_config : Sign.t -> config = fun sign ->
+  let builtin name = Extra.StrMap.find name !(sign.sign_builtins) in
+  { symb_Skolem     = builtin "Skolem" 
+  ; symb_Axiom      = builtin "Axiom" 
+  ; symb_Formula    = builtin "Formula" 
+  ; symb_imp        = builtin "imp"  
+  ; symb_forall     = builtin "forall"
+  ; symb_proof      = builtin "proof" }
+
 (** [subst_inv fu x t] replaces all the subterms [fu] by a fresh variable [x] in
```

Comment:
by a fresh variable [x] -> by [x]

Reply:
after what is implemented, this is not all the subterms [fu] but all the subterms equivalent to [fu]